balboa-agent/src/main/java/com/socrata/balboa/agent/MetricConsumer.java

```diff
@@ -147,7 +157,6 @@ public Record(String entityId, String name, Number value, long timestamp, Metric
             while (true) {
                 Map<String, String> record = grovel(stream);
                 if (record == null) {
-                    log.error("Unable to process records from {}", filePath);
                     break;
```

Comment:
I am struggling to understand what the big while true loop with the label `outer` is doing.  I am tempted to remove it to identify what it is doing.